common/cmake/do-build-helpers.cmake

```diff
@@ -104,3 +104,34 @@ macro (add_do_version_lib target_name maj_min_patch_ver)
     # CMake requires us to specify the binary dir when the source dir is not a child of the current dir
     add_subdirectory(${do_project_root_SOURCE_DIR}/common ${CMAKE_CURRENT_BINARY_DIR}/common)
 endmacro ()
+
+if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
+    # Ensure that objcopy is present
+    find_program(OBJCOPY objcopy)
+    if (OBJCOPY STREQUAL "OBJCOPY-NOTFOUND")
+        message(FATAL_ERROR "objcopy not found")
+    endif()
+endif(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
+
+# From https://github.com/dotnet/coreclr/pull/3872/files
+# For minsizerel build, split unneeded symbols from target binary file and into a separate .dbg file.
```

Comment:
>minsizerel [](http://example.com/codeflow?start=6&length=10)

Why are symbols present in the minsizerel binary? There's a seperate CMAKE_BUILD_TYPE, RelWithDebInfo, I would have assumed both Release and MinSizeRel would have automatically excluded symbols

Reply:
Don't know. That's how gcc works. I should mention that it is stripping more than just debug symbols. I didn't dig to find exactly what is stripped (will do that when there's time).